contracts/WhitelistedCommitment.sol

```diff
@@ -89,35 +87,42 @@ contract WhitelistedCommitment is AccessControlled, AccessRoles, PublicCommitmen
             curve.burnNeumark(neumarks);
         }
     }
-    event Debug(uint256 value);
-    /// overrides base class to compute neumark reward for fixed price investors
+
+    /// overrides base class to compute neumark reward for ordered whitelistfix investors
     function giveNeumarks(address investor, uint256 eth, uint256 euros)
         internal
         returns (uint256)
     {
```

Comment:
There should be preconditions like `require(euros == convertToEur(eth))` on start. But really, we should drop the `euros` argument to prevent this class of errors.

Reply:
It looks like the function arguments are redundant. We now have an extra requirement at the start of the function that the eth and euro parameters are correct with each other.

Reply:
this is a good catch! I simply forgot to remove euros parameter